metadata.json

```diff
@@ -2,7 +2,7 @@
   "author": "Vox Pupuli",
   "license": "Apache-2.0",
   "name": "puppet-r10k",
-  "version": "8.0.1-rc0",
+  "version": "8.1.0",
```

Comment:
According to this pull request #502, it should be a major release. Please change the version number to 9.0.0.

Reply:
That was done in previous version, when it was bumped to v8, right?